Dockerfile

```diff
@@ -11,7 +11,7 @@ ENV XDG_CACHE_HOME=/tmp/.chromium
 WORKDIR /usr/src/app
 
 RUN apk --no-cache upgrade && \
-    apk add --no-cache udev ttf-opensans unifont chromium ca-certificates dumb-init && \
+    apk add --no-cache udev ttf-opensans unifont chromium ca-certificates dumb-init libxslt libexpat libjxl && \
```

Comment:
These are dependencies included in `chromium` we shouldn't add them here explicitly. We install chromium as a dependency whenever we build this image so the vuln should be updated in the next release.